.github/workflows/maven.yml

```diff
@@ -286,6 +300,11 @@ jobs:
           if [[ ${{matrix.benchmark}} == templated ]]; then
             java -jar benchbase.jar -b tpcc -c config/mysql/sample_tpcc_config.xml --create=true --load=true --execute=false --json-histograms results/histograms.json
             java -jar benchbase.jar -b ${{matrix.benchmark}} -c config/mysql/sample_${{matrix.benchmark}}_config.xml --create=false --load=false --execute=true --json-histograms results/histograms.json
+          # For anonymization, we load tpcc and anonymize a single table. The workload itself is not executed
```

Comment:
Does the backend matter for coverage?  Right now we're only executing this for MySQL.

Reply:
Does not matter. I was waiting for feedback before supporting more DBs